src/main/scala/org/hammerlab/guacamole/reference/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.hammerlab.guacamole
+
+package object reference {
+  type ContigSequence = Seq[Byte]
```

Comment:
I think this is reasonable, but another option would be to just make it a trait and spell out that it implements `apply`, `slice`, and `length`. To me the latter would be a little clearer but I'm not super opposed to this solution (although if you do stick with this implementation maybe make it an `IndexedSeq`? I think it wouldn't matter as the code is now but I wouldn't want it to ever be possible to end up with a reference genome as a `List`)